sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -258,6 +258,11 @@ abstract class RapidsMeta[INPUT <: BASE, BASE, OUTPUT <: BASE](
    */
   def canPartsBeReplaced: Boolean = childParts.forall(_.canThisBeReplaced)
 
+  /**
+   * Return true if this supports columnar execution (ie is replaced)
```

Comment:
The comment here implies this is equivalent to `canThisBeReplaced` and it doesn't help that the implementation states it as well.
```suggestion
   * Return true if the resulting node in the plan will support columnar execution
```